dashboard/src/deployment/DeploymentList.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { apiGet } from '../api/api.js';
+import DeploymentRow from './DeploymentRow.js';
+import { Table } from 'semantic-ui-react';
+//import logo from './logo.svg';
+//import './App.css';
+
+class DeploymentList extends Component {
+  constructor() {
+    super();
+    this.state = {items:[]};
+  }
+
+  async componentDidMount() {
+    this.reloadDeployments();
+  }
+
+  async reloadDeployments() {
+    const result = await apiGet('/api/deployment');
+    this.setState({items:result.deployments});
+  }
+
+  render() {
+    setTimeout(this.reloadDeployments.bind(this), 5000);
+    const items = this.state.items;
+    if (items.length === 0) {
+      return (<div>No deployments</div>);
+    }
+    return (
+      <Table striped celled>
```

Comment:
Again, I'd like to keep the presentational part in a presentational component to make it easier to swap out later.